integration-cli/docker_cli_commit_test.go

```diff
@@ -116,9 +116,9 @@ func (s *DockerSuite) TestCommitChange(c *check.C) {
 		"--change", "ENV test 1",
 		"--change", "ENV PATH /foo",
 		"--change", "LABEL foo bar",
-		"--change", "CMD [\"/bin/sh\"]",
+		"--change", `"CMD [""/bin/sh""]"`,
```

Comment:
ping @dnephin 
In a shell (`zsh`), with docker 1.11 : `λ docker commit --change 'CMD ["/bin/sh"]' 830` works fine, with current master `docker commit --change 'CMD ["/bin/sh"]' ef9` returns *"invalid argument "CMD [\"/bin/sh\"]" for --change: line 1, column 5: bare " in non-quoted-field"*.